README.md

```diff
@@ -55,7 +58,7 @@ Now you can talk to your bot in your mattermost client!
 
 A chat bot is meaningless unless you can extend/customize it to fit your own use cases.
 
-To write a new plugin, simply create a function decorated by `mattermost_bot.bot.respond_to` or `mattermost_bot.bot.listen_to`:
+To write a new plugin, simply create a python file with function decorated by `mattermost_bot.bot.respond_to` or `mattermost_bot.bot.listen_to in the ".../mattermost_bot/plugins" directory `:
```

Comment:
I think you are thinking about modularizing your bot which is probably a better approach but it introduces a lot of redundancies e.g., even if you write a new plugin in a new project, you still have to modify the site-packages source code i.e., modify "mattermost_bot_settings.py" file every time you write a new plugin which is not very modular.

But if you ask them to develop a new plugin and just placing it in the plugins directory, there is no need to modify the "site-packages" source.